.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_size = 1
```

Comment:
this means all JavaScript, CSS & HTML, etc.. indentation will be 1 space only which is bit unusual maybe we should ask @domenic in #18 about his preference.

Reply:
Yeah. Currently, most (if not all) HTML files use 1 space only; JavaScript files and inline `<script>` elements use 2 spaces; `.htaccess` files are not indented consistently. And I don't think we need to worry about the indentation of SVG files in this case.

I'll wait for Domenic's reply and adjust the PR if necessary.

Reply:
1 space is preferred for HTML (or SVG). I guess to be consistent we should also do it for CSS and script and .htaccess, especially since we don't use file extensions so differentiating would be tricky (see #8).

I think we should fix those files to be 1 space while merging this if you're up for it. Otherwise I think adding .editorconfig will just cause confusion since it will cause peoples' text editors to start being inconsistent with the indentation inside a file.

Reply:
Can we have two spaces for .js ?

Reply:
@zcorpan Done.